osu.Game.Rulesets.Mania.Tests/Editor/ManiaSelectionBlueprintTestScene.cs

```diff
@@ -1,31 +1,55 @@
 // Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
 // See the LICENCE file in the repository root for full licence text.
 
+using System.Collections.Generic;
 using osu.Framework.Allocation;
 using osu.Framework.Graphics;
-using osu.Framework.Timing;
+using osu.Framework.Graphics.Containers;
+using osu.Game.Rulesets.Edit;
+using osu.Game.Rulesets.Mania.Beatmaps;
 using osu.Game.Rulesets.Mania.UI;
+using osu.Game.Rulesets.UI;
+using osu.Game.Rulesets.UI.Scrolling;
 using osu.Game.Tests.Visual;
-using osuTK.Graphics;
 
 namespace osu.Game.Rulesets.Mania.Tests.Editor
 {
-    public abstract class ManiaSelectionBlueprintTestScene : SelectionBlueprintTestScene
+    [Cached(typeof(IPlayfieldProvider))]
+    public abstract class ManiaSelectionBlueprintTestScene : SelectionBlueprintTestScene, IPlayfieldProvider
     {
-        [Cached(Type = typeof(IAdjustableClock))]
-        private readonly IAdjustableClock clock = new StopwatchClock();
+        protected override Container<Drawable> Content => blueprints ?? base.Content;
```

Comment:
What's the null fallback here for? `blueprints` can't possibly be null, can it?

Reply:
It is null when used during the base constructors.

Reply:
I would hope nobody is reading this fallback value, given that it may cause children to be added to a different container than expected, but that said other test scenes seem to be doing similar things so...